src/services/fileStorage/strategies/awsS3.js

```diff
@@ -25,11 +25,13 @@ try {
 
 const AbstractFileStorageStrategy = require('./interface.js');
 
-const createAWSObject = (schoolId) => {
-	if (!awsConfig.endpointUrl) throw new Error('AWS integration is not configured on the server');
+const createAWSObject = (schoolId, configIndex) => {
+	if (!awsConfig[0]) throw new Error('AWS integration is not configured on the server');
 
-	const config = new aws.Config(awsConfig);
-	config.endpoint = new aws.Endpoint(awsConfig.endpointUrl);
+	configIndex = (awsConfig[configIndex] ? configIndex : 0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment to function parameter 'configIndex'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5222818)